apps/pie-microsite/src/assets/styles/components/_contentPage.scss

```diff
@@ -79,6 +79,10 @@
         margin-top: f.spacing(e);
     }
 
+    h2 {
+        @include f.font-size('heading-l');
```

Comment:
It looks like the default `h2` size is wrong in fozzie (which is why you need to add these styles, as the microsite pulls in fozzie). We likely need an update ticket to change this in fozzie so it's up-to-date with the [latest foundations](https://www.figma.com/file/8LrMTRRHfQYSRZrK4cLoKs/%E2%9A%A1%EF%B8%8F%F0%9F%A7%8A-PIE-2.0-Foundations-%E2%80%93-Light-%F0%9F%A7%8A%E2%9A%A1%EF%B8%8F?node-id=5437%3A20693&t=o9F7a6rkwdFwU7jC-4).

Shouldn't hold up this ticket, but good that this has brought inconsistency up 👍

Reply:
I think @jamieomaguire has created a ticket for this update as well. we will relook at it in our next refinement hopefully. 💯